src/actions/authenticationActions.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
+import toastr from "toastr";
+import * as types from "./actionTypes";
+import AuthenticationAPI from "../api/authenticationAPI";
+
+export const loginByEmailActionSuccess = (auth = {}) => ({
+  type: types.LOGIN_BY_EMAIL_SUCCESS,
+  payload: auth
+});
+
+export const loginByEmailActionFailure = (content = {}) => ({
+  type: types.LOGIN_BY_EMAIL_FAILURE,
+  payload: content
+});
+
+export const login = userDetails => (dispatch) => {
+  AuthenticationAPI.login(userDetails).then((response) => {
+    if (response.success) {
+      toastr.success("Login Successful");
+      localStorage.setItem("user", JSON.stringify(response.content));
```

Comment:
what does the `response.content` have ? cause all you need to put in the localStorage is token

Reply:
@denomash and @bett requires specific user details from the local storage so I decided to save all user details to the storage

Reply:
No,  save the token on the localstorage and let them access the other user details on the store, its not good practice to put all users data on localstorage.

Reply:
i will implement the changes